frontend/src/features/whats-new/utils/utils.ts

```diff
@@ -4,28 +4,22 @@ import { UserDto } from '~shared/types/user'
 
 import { FeatureUpdateList } from '../FeatureUpdateList'
 
-export interface shouldShowLatestFeatureUpdateNotificationProps {
-  user: UserDto | undefined
-  isLoading: boolean
-}
+export const getShowLatestFeatureUpdateNotification = (
+  user: UserDto | undefined,
+): boolean => {
+  if (user?.flags?.lastSeenFeatureUpdateDate) {
+    const sortedFeatureUpdateList = FeatureUpdateList.sort(
+      (featureUpdateA, featureUpdateB) =>
+        Date.parse(featureUpdateB.date) - Date.parse(featureUpdateA.date),
+    )
+    const lastSeenUserFeatureUpdateDate = new Date(
+      user?.flags?.lastSeenFeatureUpdateDate,
+    )
```

Comment:
is there a need for this conversion? is the flag not stored as a date type? if this is due to the serialization due to client-server boundary, we have a [`transformAllIsoStringsToDate`](https://github.com/opengovsg/FormSG/blob/bc928f713e717d9137a55f2acff5f025fde42d9d/frontend/src/utils/date.ts#L125) transformation function.

Reply:
Nope there is no need for conversion